lasagne/__init__.py

```diff
@@ -9,3 +11,6 @@
 from . import regularization
 from . import updates
 from . import utils
+
+
+__version__ = pkg_resources.get_distribution("Lasagne").version
```

Comment:
Maybe add a `del pkg_resources` afterwards, so we don't export `lasagne.pkg_resources`? I like clean namespaces.